plugins/ltac2/tac2env.ml

```diff
@@ -308,11 +308,23 @@ let ltac1_prefix =
 
 (** Generic arguments *)
 
-let wit_ltac2 = Genarg.make0 "ltac2:tactic"
-let wit_ltac2_val = Genarg.make0 "ltac2:value"
+module Ltac2in1 = struct
```

Comment:
This should probably go into Tac2expr instead, and also I'm not a huge fan of the module wrapper.

Reply:
changed